components/home/CosmosMigrationBanner.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="cosmos-migration-banner">
+    <div class="cosmos-migration-banner__left-col">
+      <div class="cosmos-migration-banner__title">
+        {{ $t('CosmosMigrationBanner.title') }}
+      </div>
+      <div class="cosmos-migration-banner__subtitle">
+        {{ $t('CosmosMigrationBanner.subtitle') }}
+      </div>
+      <div class="cosmos-migration-banner__action-button">
+        <md-button
+          class="md-likecoin lc-gradient-2 lc-font-size-20 lc-font-weight-600 shadow"
+          :href="migrationURL"
+        >{{ $t('CosmosMigrationBanner.actionButtonText') }}</md-button>
+      </div>
+    </div>
+    <div class="cosmos-migration-banner__right-col" />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { getMigrationSiteURL } from '@/util/api/api';
+
+export default {
+  name: 'cosmos-migration-banner',
+  computed: {
+    ...mapGetters([
+      'getUserInfo',
```

Comment:
this component might be cleaner if we can just pass `user` string as prop, and call `getUserInfo ` from  parent instead